sdk/storage/Azure.Storage.Common/src/GeoRedundantReadMode.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Storage
+{
+    /// <summary>
+    /// Pattern for retrying a read request between geo-redundant primary and secondary endpoints.
+    /// </summary>
+    public enum GeoRedundantReadMode
```

Comment:
Would it make any sense to include `None` as a default value?

Reply:
The SDK is already in PrimaryThenSecondary mode today, and just doesn't bother if the secondary endpoint wasn't provided. I don't know what None would help convey here.